### ISSUE TITLE
Problem in searching with space using chosen

### ISSUE THREAD

**Issue by gunjanpatel** [joomla/joomla-cms#7986]
#### Steps to reproduce the issue
+ Create a category having space in name. For example: "Events and Forms"
+ Make sure you have more than 10 categories, if not try to copy it.
+ Go to article create/edit page, In category selection try to search "event and "

![screen shot 2015-09-30 at 05 00 45](http://issues.joomla.org/uploads/1/483cba7e71c100824543e0f8b9d35905.png)

+ Now, close the article edit page and go to article listing page. Select any article and click on batch button from toolbar and try to search "event and " like below and notice it is working.

![screen shot 2015-09-30 at 05 04 18](http://issues.joomla.org/uploads/1/f59d9073b5ee1e1f80d74c1d1e8980f8.png)

#### Expected result

It should looks like below in article edit page.
![screen shot 2015-09-30 at 05 05 36](http://issues.joomla.org/uploads/1/6435a326b0a44f88ce045cd4b6883849.png)

#### System information (as much as possible)
I have tested it in FF41.0 Ubuntu. But I believe it's not dependent on browser or OS.

**Comment by gunjanpatel** [comment]
This issue can be fixed by using,
```php
JHtml::_('formbehavior.chosen', 'select', null, array('search_contains' => true));
```
but not sure we should fix it in general by setting "search_contains" as a general options.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7986">issues.joomla.org/joomla-cms/7986</a>.</sub>

**Comment by javigomez** [comment]
I was able to reproduce the issue. See the following 1 minute video (with audio): https://www.youtube.com/watch?v=h5N5AD8hAls

Important note. As mentioned in the issue description you need to have more than 10 categories. Otherwise you will not see the Searchbox to search Categories by name.

Seems to be a Javascript issue in Chosen Selector not being able to read spaces.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7986">issues.joomla.org/joomla-cms/7986</a>.</sub>

**Comment by javigomez** [comment]
Linking possible solution pull request: https://github.com/joomla/joomla-cms/pull/7987 #7987<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7986">issues.joomla.org/joomla-cms/7986</a>.</sub>

**Comment by gunjanpatel** [comment]
Closing this issue now, as PR is available at #7987 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7986">issues.joomla.org/joomla-cms/7986</a>.</sub>
